### PR TITLE
Exclude failing tests on appveyor

### PR DESCRIPTION
**Pull request by kevinwlau**
Fix for #497 until we investigate the root cause

Changes:
- Move PollingDevTest to use gradle 4.10
- Exclude TestLoose* tests (3 of the 4 fail on checking for `commons-text-1.1.jar`

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ before_build:
 build_script:
     - cmd: |
         set GRADLE_OPTS="-Dorg.gradle.jvmargs='-XX:MaxMetaspaceSize=512m'"
-        gradlew.bat clean install check -Ptest.exclude="**/*15*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
+        gradlew.bat clean install check -Ptest.exclude="**/*15*,**/Polling*,**/*estLoose*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
```
Comment (cherylking) [inline appveyor.yml:38]: Curious why not **/TestLoose*? Did you leave the T off on purpose? Also, can you open an issue specifically listing the tests that this PR excluded and need to be investigated (and reference this PR in that issue)?

File: appveyor.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ before_build:
...
         gradlew.bat wrapper --gradle-version 4.10
         gradlew.bat check -Ptest.include="**/*15*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
```
Comment (cherylking) [inline appveyor.yml:40]: One more thought...can you see if any of the tests you are excluding actually work with this older gradle wrapper?

File: appveyor.yml (hunk 1 above)
Comment (kevinwlau) [reply appveyor.yml:38]: For some reason, I seemed to have problems in the past getting it to filter correctly without the first wildcard. I can try again to make it cleaner.
#497 already lists the issues and shouldn't be auto-closed by this PR.

File: appveyor.yml (hunk 2 above)
Comment (kevinwlau) [reply appveyor.yml:40]: Seems like the TestLoose* still fails when using gradle 4.10, and the PollingDevTest ran fine until it errored on cleanup.

Will see if any of the other jobs or reruns complete with a success on PollingDevTest.

**Pull request merged**